git-examples.md

```diff
@@ -72,15 +72,15 @@ There you should see the message that a new branch has been pushed to your forke
 ```sh
 cd Firmware
 ```
-* Make a new branch that describes new feature:
+* Make a new branch that describes the new feature:
 ```sh
 git checkout -b feature_branch
 ```
-* Go go submodule subdirectory
+* Go to submodule subdirectory
 ```sh
 cd <path to submodule>
 ```
-* PX4 submodule might not necessarilty point to the newest commit. Therefore, first checkout out master and pull the newest upstream code.
+* PX4 submodule might not necessarily point to the newest commit. Therefore, first checkout master and pull the newest upstream code.
```

Comment:
I am sorry. the steps that I wrote have to be done if you like to have a submodule update in the Firmware.
But anyway, the way I understand it (and I will add it to dev guide afterwards) is as follow (try to explain it with a simple example):
Assume we are working in the Firmware. At some point we realize that there is a bug in the submodule sitl_gazebo. In order to fix it, one approach is to go to the sitl_gazebo directory and do a pull such that we have the newest commits. Then we fix the bug, make a new branch that describes the bug, then do an add and a commit, and finally push it to the submodule origin. Then we go online and do a PR for that bug_fix. 
Once the bug_fix is in the submdolue master, you need to do a PR for submodule update. To do so you have to follow the steps I wrote in the guide. I will update it now